frontends/infinite-corridor/src/api/learning-resources/hooks.test.tsx

```diff
@@ -138,54 +138,71 @@ test("useDeleteUserList invalidates all resource queries", async () => {
   expect(before.listing.data).not.toEqual(after.listing.data)
 })
 
-test("useAddToUserListItems invalidates userlist details and listing", async () => {
-  const { wrapper, spies } = setup()
-
-  const list = factories.makeUserList()
-  const resource = factories.makeCourse()
-  const modifiedAddedResource = {
-    ...resource,
-    lists: resource.lists.concat({} as ListItemMember)
+test.each([
+  {
+    list: factories.makeUserList({ id: 123 }),
+    url:  urls.userList.itemAdd(123)
+  },
+  {
+    list: factories.makeStaffList({ id: 456 }),
+    url:  urls.staffList.itemAdd(456)
   }
+])(
+  "useAddToListItems invalidates userlist details and listing",
```

Comment:
Minor nitpick. maybe change to "useAddToListItems invalidates userlist/stafflist details and listing".  Or add a `mode` variable and apply it here as in another test below in this PR:  "useAddToListItems invalidates ($mode) details and listing"